DistributionModels/weibull.py

```diff
@@ -2,14 +2,18 @@
 import torch
 
 class weibull:
-    def __init__(self, saved_model=None):
+    def __init__(self, translateAmountTensor=1,saved_model=None):
```

Comment:
If `translateAmountTensor` is a float/int rather than a tensor this variable should be renamed.

Reply:
Also, please stick to the pythonic way of naming variables, i.e., use `translate_amount_tensor` instead of the camelcase. Also, please **do not change** the order of parameters that are already there, otherwise a call to `weibull(model)` will not work any longer.